pkg/recipe/recipe.go

```diff
@@ -59,7 +59,8 @@ func (re *Recipe) Render(engine RenderEngine) error {
 	idx := 0
 	for filename, content := range files {
 		sum := sha256.Sum256(content)
-		re.Files[filename] = File{Content: content, Checksum: fmt.Sprintf("sha256:%x", sum)}
+		file := File{Content: content, Checksum: fmt.Sprintf("sha256:%x", sum)}
+		re.Files[filename] = file
```

Comment:
Just curious, was this change because it is more idiomatic?

Reply:
Ah no, actually it's just leftovers from 923b0e745e1dfb84613527c932254ea8f74615b3 and should be put back the way it was.